fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -1713,6 +1713,13 @@ def _post_backward_hook(self, param: Parameter, *unused: Any) -> None:
 
         # Switch to FP32 shard after backward.
         self._use_fp32_param_shard([param])
+        if self.mixed_precision and self.fp32_reduce_scatter:
```

Comment:
Currently for fp8, we do not use `mixed_precision`, so we should remove this. 

Only check 

`if self.fp32_reduce_scatter:`

Reply:
Addressed the comment.